lib/PEX.ts

```diff
@@ -290,7 +295,8 @@ export class PEX {
     version?: PEVersion;
     error?: string;
   } {
-    const data = { presentation_definition: presentationDefinition };
+    const presentationDefinitionCopy: PresentationDefinitionV1 = JSON.parse(JSON.stringify(presentationDefinition));
```

Comment:
What about V2?

Reply:
BTW we might want to introduce a new type anyway, like we have done in other areas. That would help here as well.

type PresentationDefinition = PresentationDefinitionV1 | PresentationDefinitionV2

Reply:
created